services/forms/user_form.go

```diff
@@ -118,7 +119,8 @@ func IsEmailDomainListed(list []string, email string) bool {
 	domain := strings.ToLower(email[n+1:])
 
 	for _, v := range list {
-		if strings.ToLower(v) == domain {
+		matched, _ := filepath.Match(strings.ToLower(v), domain)
```

Comment:
Using a I/O function for this is incorrect(and has unexpected side-effects). We already use wildcarding trough the code via the `github.com/gobwas/glob` package, please adjust the code to use this package.

Reply:
thankyou for the feedback. I've made the change to use glob.